tools/mo/openvino/tools/mo/front/tf/FifoQueueDequeueCut.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+from collections import defaultdict
+
+from openvino.tools.mo.front.common.replacement import FrontReplacementPattern
+from openvino.tools.mo.front.extractor import add_input_ops
+from openvino.tools.mo.front.output_cut import OutputCut
+from openvino.tools.mo.front.user_data_repack import UserDataRepack
+from openvino.tools.mo.graph.graph import Graph
+from openvino.tools.mo.middle.passes.convert_data_type import np_data_type_to_precision, SUPPORTED_DATA_TYPES
+from openvino.tools.mo.utils.error import Error
+
+
+class FifoQueueDequeueCut(FrontReplacementPattern):
+    """
+    Cuts FifoQueue -> QueueDequeue pattern
```

Comment:
Please replace: FifoQueue  -> FIFOQueue
Also, please add a comment:
This transformation relies on output shapes and types extracted from QueueDequeue node. In the meantime, the transformations FIFOQueue and QueueDequeueManyV2 expects output shapes and types extracted from FIFOQueue node